regression/goto-diff/java-scope-change2/test.desc

```diff
@@ -0,0 +1,14 @@
+// Known bug as currently no difference is reported.
+// At least foo should report as changed as the variable within the class has a scope change (public to private).
```

Comment:
Access modifiers are not compared at the moment. I'll fix that.

Reply:
Thanks.